hotosm/galaxy-ui
# Sortable table columns via mapathon and user report pages.

**What does this PR do?** 
This PR adds functionality allowing users to sort different table columns either by ascending or descending order on both the mapathon detailed and user group reports pages. 

In use:
- [React-Table](https://react-table-v7.tanstack.com/)
- [tailwind components](https://tailwind-elements.com/docs/standard/components/tables/)

**How to test:**
- Pull the branch `git fetch origin && git checkout feature/add-table-functionality`. 
- There's a new dependency in use, i.e. react-table. Hence, we install packages with: `npm i` 
- Start the server with `npm start`
- Go to `http://127.0.0.1:3000/mapathon-report/detailed` and submit form data. 
- Click on different table column headers to sort either by ascending or descending order when the results table is displayed.

**Screenshot:** 
![Screenshot 2022-04-15 at 07 37 28](https://user-images.githubusercontent.com/31903212/163519219-4c72eb69-4908-4557-9ca0-fa1c121571b9.png)

**Related issues:** 
- Fix #70
